File: e2e/sdk-matrix/ruby/main.rb
require "sentry-ruby"

Sentry.init do |config|
  config.dsn = ENV.fetch("SPICYTRACK_DSN")
  config.environment = "sdk-matrix"
  config.release = "sdk-ruby@6.7.0"
end

begin
  raise "Real Ruby SDK compatibility probe"
rescue RuntimeError => error
  Sentry.capture_exception(error)
end
Sentry.get_current_client.flush

File: e2e/sdk-matrix/python/main.py
import os
import sentry_sdk

sentry_sdk.init(
    dsn=os.environ["SPICYTRACK_DSN"],
    environment="sdk-matrix",
    release="sdk-python@2.66.1",
    send_default_pii=False,
)
try:
    raise RuntimeError("Real Python SDK compatibility probe")
except RuntimeError as error:
    sentry_sdk.capture_exception(error)
sentry_sdk.flush(timeout=10)
